XbSymbolDatabase.c

```diff
@@ -202,24 +202,32 @@ void XbSymbolBypassRevLimit(bool bypass_limit)
     bStrictRevisionLimit = !bypass_limit;
 }
 
-void XbSymbolOutputMessage(xb_output_message mFlag, const char* message)
+// Intended for send the message as-is without formating.
+// (To avoid corrupted string when perform a bad coding.
```

Comment:
If this happens, the calling code should be fixed - falling back to a "safer" but possibly erroneous print shouldn't even be an option.